src/main/java/games/strategy/engine/data/CompositeRouteFinder.java

```diff
@@ -15,7 +16,7 @@
   private static final Logger s_logger = Logger.getLogger(CompositeRouteFinder.class.getName());
 
   private final GameMap m_map;
-  private final HashMap<Match<Territory>, Integer> m_matches;
+  private final Map<Match<Territory>, Integer> m_matches;
```

Comment:
Are you sure changing this member variable type doesn't break compatibility of save games or lobby?